metrics/README.md

```diff
@@ -0,0 +1,62 @@
+# Clear Containers metrics tests
+
+This is a collection of scripts designed to measure performance and responsiveness
```

Comment:
Let's change the beginning and add a "the" before performance:
This section contains a collection of scripts designed to measure the performance...